src/Api/Billing/Public/Controllers/OrganizationsController.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System.Net;
+using Bit.Api.Models.Public.Response;
+using Bit.Core.AdminConsole.Entities;
+using Bit.Core.Context;
+using Bit.Core.OrganizationFeatures.OrganizationSubscriptions.Interface;
+using Bit.Core.Repositories;
+using Bit.Core.Services;
+using Bit.Core.Utilities;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using OrganizationSubscriptionUpdateRequestModel = Bit.Api.Billing.Public.Models.OrganizationSubscriptionUpdateRequestModel;
+
+namespace Bit.Api.Billing.Public.Controllers;
+
+[Route("public/billing-organization")]
```

Comment:
The value for this route should be `public/organization` in keeping with the AC public controller of the same name. The API user shouldn't care how we set up our teams and code ownership